### DISCUSSION TITLE
Delete unused blocks

### DISCUSSION THREAD

**Discussion by Thatsnice1938** [ScratchAddons/ScratchAddons#6202] [Ideas] [answered]
In the Developer tools addon, there is enhanced clean-up blocks+ that remove all unused orphaned reporter blocks. I suggest having it remove regular blocks too.

**Comment by kokofixcomputers** [comment, marked as answer]
like blocks that are not connected, just lying in the code area? if you meant that i would like that too

**Comment by Thatsnice1938** [comment]
Yes
